### PR TITLE
fix:The Build Task for i18n to res folder not working properly

### PR DESCRIPTION
**Pull request by angrezichatterbox**
### Contributor checklist

-   [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
-   [x] I have tested my code with the `./gradlew lintKotlin detekt test` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Data/blob/main/CONTRIBUTING.md#testing)

---

### Description

The gradle task for transferring files from i18n to the res folder for localization was not transferring files, which caused the i18n update to not be reflected in the res folder. 

### Related issue

-   #243

### CODE REVIEW

**Review by MahmoudMabrok — APPROVED** [review]

File: app/build.gradle.kts [hunk 1]
```diff
@@ -201,18 +201,23 @@ dependencies {
 }
 
 tasks.register<Copy>("moveFromi18n") {
-    val locales = listOf("de", "es", "sv", "en-US")
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    destinationDir = file("src/main/res")
 
+    val locales = listOf("de", "es", "sv", "en-US")
```
Comment (andrewtavis) [inline app/build.gradle.kts:207]: Do we need to expand this with the other localizations that are included in Scribe-i18n?

Comment (MahmoudMabrok) [reply app/build.gradle.kts:207]: val  locales = file("src/main/assets/i18n/Scribe-i18n/values").listFiles()
        .findAll { it.isDirectory() }
        .collect { it.name }
        
could we use something like this

**Comment by andrewtavis** [comment]
Appreciate the help, @MahmoudMabrok! Your version of getting the locales worked great, and from there we needed to add them all to the UI. This has been done by hand so far, but maybe we can figure out a way to load those dynamically as well 😊

**Review by andrewtavis — APPROVED** [review]
Thanks for the quick fix here, @angrezichatterbox, and also for the collaboration to get it all working, @MahmoudMabrok! 😊

**Pull request merged**